dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/utils/JacksonSerializer.java

```diff
@@ -31,7 +39,13 @@
 	 * @return byte array
 	 */
 	public static <T> byte[] serialize(T obj)  {
-		String json = JSON.toJSONString(obj);
+        String json = "";
```

Comment:
In original logic ,if obj can't format to json ,exceptions throw. However current method return empty string ,which maybe change the logic of the invoker. You'd better make sure about this will not have any impact on invoker.

Reply:
toJSONString will not throw exceptions